src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -467,6 +467,18 @@ function Get-MarketplaceImage
     return $imgs;
 }
 
+function Get-FirstMarketPlaceImage
```

Comment:
where is this used?

Reply:
In the test file below this. It removes like 99% of the file because it doesn't have recursively increasing piping.